src/container/cli.c

```diff
@@ -2415,8 +2416,11 @@ cont_req_cleanup(enum creq_cleanup_stage stage, bool free_tpriv, struct cont_req
 	case CLEANUP_RPC:
 		crt_req_decref(args->cra_rpc);
 	case CLEANUP_TASK_PRIV:
-		if (free_tpriv)
+		if (free_tpriv) {
 			D_FREE(args->cra_tpriv);
+			args->cra_tpriv = NULL;
```

Comment:
D_FREE() will set NULL, you don't have to do it.

Reply:
will fix